src/main/java/dataStructures/hashSet/openAddressing/README.md

```diff
@@ -6,6 +6,37 @@ A hash collision is resolved by <b>probing<b>, or searching through alternative
 the array (the probe sequence) until either the target element is found, or an unused array slot is found,
 which indicates that there is no such key in the table.
 
+## Implementation Invariant
```

Comment:
looks good. just restructure to the following:

# HashSet (Open-Addressing)

## Background
 ... knowledge on hashset / intro

### Implementation Invariant
...
bold the key sentence here!

### Probing Strategies
...

## Complexity Analysis
...

## Notes
any other misc u wanna put

Reply:
resolved in 19a45ef3f87380d53877fc4d26a76c5a9ed46628